Formula/pgbadger.rb

```diff
@@ -16,6 +16,7 @@ class Pgbadger < Formula
 
   def install
     system "perl", "Makefile.PL", "DESTDIR=#{buildpath}"
+    inreplace "Makefile", "sed -i '", "sed -i '' '"
```

Comment:
This issue needs to be reported and fixed by upstream.

Reply:
Will do

Reply:
Change merged upstream. Now awaiting a release:

https://github.com/darold/pgbadger/pull/438